WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -768,6 +769,7 @@ private void setupAddMenuPopup() {
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 adapter.notifyDataSetChanged();
 
+                AppLockManager.getInstance().setExtendedTimeout();
```

Comment:
This doesn't fix the problem for me. This code runs after the user clicks one of the menu items, but the pin lock screen appears after clicking the plus icon and granting permissions - which is before the user has selected from the menu.